Orchestrator.py

```diff
@@ -83,14 +84,40 @@ def _sendDownstreamCommands(self):
         Send the next heading and speed commands to the robots
         '''
 
+        log.debug(f'dotbotsView -> {self.dotBotsView}')
+
+        # update dotbot positions before giving new heading and speed
+        for idx, dotbot in self.dotBotsView.items():
+            if dotbot['bump'] == False:
```

Comment:
Remove this check

Reply:
but in that case it assumes the dotbot kept moving in that direction for the set amount of time, but in reality it stopped before getting there. the orchestrator will assume it's inside the obstacle, no?

Reply:
Change heading when receiving a bump command, not here

Reply:
so we go back to only commanding robots to move when they bump not every second?